### PR TITLE
Dev/latency bag fix

### PR DESCRIPTION
**Pull request by josephvanpeltkw**
adds a new ImageMetadata message that stores the image timestamp versus when the ImageMetadata was published. This allows the latency tracking node to lookup when an image was processed (and fixes the latency calculation for bagged images).

### CODE REVIEW

File: ansible/roles/provision-files/vars/main.yml [hunk 1]
```diff
@@ -71,15 +71,15 @@ girder_file_downloads:
       sha512: e6b382f5ba3d4d7f17d29caa3f6ea06dd5db24fee3bf59b12ff748d58b1b064c374741c805ee7c07a2a6950761a38e5398f3707ed3e83d7d53218972c57d12a2
       dest: "{{ stage_dirs.object_detector }}/m2_det.pt"
     # Activity classifier
-    - file_id: 671bff2983e1acbfd87e723f
-      sha512: 5165219f60e3a160a8ec51f0446db4ec23ed70c399df66f482a233566e177d1f8cf044fc6ccc5227f5c748cbe6d2f17143109e791a139dfc2225d75de67829b2
```
Comment (josephvanpeltkw) [inline ansible/roles/provision-files/vars/main.yml:75]: https://data.kitware.com/#item/671bff2983e1acbfd87e723e

Comment (josephvanpeltkw) [reply ansible/roles/provision-files/vars/main.yml:75]: New: https://data.kitware.com/#item/673f69bef4ba555b2ab8ae14

File: ansible/roles/provision-files/vars/main.yml [hunk 2]
```diff
@@ -122,15 +122,15 @@ girder_file_downloads:
       sha512: 7183385f8eaca85997725a107a76034de2bd4a59c1434b4bdb7c1ac8931cf4b68a53f6e736734643386364b9f0856de795a14965b6a02bc5eb5891252e6a73c9
       dest: "{{ stage_dirs.object_detector }}/r18_det.pt"
     # Activity classifier
-    - file_id: 672cf299ca1110761bd32ffd
-      sha512: 0bfd4c5a1f254120758b9802eefef8933bfebf80837bd61445c3c26228edaefa34f0ed81178670538426983a93d60198f1be82a767e350b26d4fb3d081972b0d
```
[the remark this answers is not in the document]
Comment (josephvanpeltkw) [reply ansible/roles/provision-files/vars/main.yml:126]: New: https://data.kitware.com/#item/673f6e62f4ba555b2ab8ae34

File: ansible/roles/provision-files/vars/main.yml [hunk 3]
```diff
@@ -122,15 +122,15 @@ girder_file_downloads:
...
       dest: "{{ stage_dirs.activity_classifier }}/r18_tcn.ckpt"
-    - file_id: 672cf437ca1110761bd33009
-      sha512: d99c484071f22661425a9673ff351db90ce2fba40f2faa56f0b388deb2d0cdcc96931a2627e5aec45ab5a4624c9ab04da23b63364dd6eb1d5641a90701737006
+    - file_id: 673f6e58f4ba555b2ab8ae32
+      sha512: 09a4c97e79c34ac95c95a4c63370b928527a98b9bab233aa12e4b14cfa95e0bb5a09bad90d19715ab656d17281ba5fbaa719647a5545bf2447ae584da5bd8d6a
       dest: "{{ stage_dirs.activity_classifier }}/r18_config.yaml"
     # Global Step predictor model
-    - file_id: 66464bf9687336214e7cdeae
-      sha512: bc7884c258cb8704372dd69a3e7d999eaf70d716d271982077c7216ef27ab01219ef1e488f6f9669e11a7e8aa6ffb9d8e07d74edc47294f90cc9e6a723f5a822
+    - file_id: 673f8cbcf4ba555b2ab8ae42
```
Comment (josephvanpeltkw) [inline ansible/roles/provision-files/vars/main.yml:132]: New: https://data.kitware.com/#item/673f8cbcf4ba555b2ab8ae41

File: ansible/roles/provision-files/vars/main.yml (hunk 1 above)
Comment (Purg) [reply ansible/roles/provision-files/vars/main.yml:75]: By "comments" I was at least meant to refer to comments in the actual file. @cameron-a-johnson is that what you meant as well, or this kind of github comment?

File: ansible/roles/provision-files/vars/main.yml (hunk 3 above)
Comment (Purg) [reply ansible/roles/provision-files/vars/main.yml:132]: Hashes look correct.

**Comment by Purg** [comment]
FWIW, this branch demoed appropriately. We need to verify that what is on the laptop is appropriately synched with the configurations such that we confirm reproducibility and then we need to tag this version.

**Comment by Purg** [comment]
Merging due to confirmed functionality during IITSEC and PI meeting.

**Review by Purg — APPROVED** [review]

**Pull request merged**